layers/+fun/emoji/packages.el

```diff
@@ -39,4 +39,21 @@
     (use-package company-emoji
       :if (configuration-layer/package-usedp 'company)
       :defer t
-      :init (setq company-emoji-insert-unicode nil))))
+      :init
+      (progn
+        (setq company-emoji-insert-unicode nil)
+        ;; From https://github.com/dunn/company-emoji/README.md for Linux, or on Mac OS X and using the Cocoa version of Emacs
+        (defun --set-emoji-font (frame)
```

Comment:
If it's only useful in gui, wouldn't adding a guard `when display-graphic-p` in the function avoid it setting unnecessarily?